integration/httpd_reload_test.go

```diff
@@ -0,0 +1,117 @@
+package integration_test
+
+import (
+	"bytes"
+	"fmt"
+	"os"
+	"path/filepath"
+	"strings"
+	"testing"
+
+	"github.com/paketo-buildpacks/occam"
+	"github.com/paketo-buildpacks/packit/v2/pexec"
+	"github.com/sclevine/spec"
+
+	. "github.com/onsi/gomega"
+	. "github.com/paketo-buildpacks/occam/matchers"
+)
+
+func testHttpdReload(t *testing.T, context spec.G, it spec.S) {
+	var (
+		Expect     = NewWithT(t).Expect
+		Eventually = NewWithT(t).Eventually
+
+		pack   occam.Pack
+		docker occam.Docker
+		source string
+		name   string
+	)
+
+	it.Before(func() {
+		pack = occam.NewPack().WithVerbose()
+		docker = occam.NewDocker()
+	})
+
+	context("when the buildpack is run with pack build", func() {
```

Comment:
and `BP_LIVE_RELOAD_ENABLED` set to true

Reply:
done